scipy/stats/_stats_py.py

```diff
@@ -9703,14 +9710,14 @@ def wasserstein_distance(u_values, v_values, u_weights=None, v_weights=None):
                 u\\
                 v\\
             \end{bmatrix}
-
+    The :math:`\text{vec}()` function denotes the Vectorization fucntion.
     The tranport plan :math:`\Gamma` is a matrix :math:`[\gamma_{ij}]` in
     which :math:`\gamma_{ij}` is a positive value representing the amount of
     probability mass transported from :math:`u` to :math:`v`. Summing over
     the rows of :math:`\Gamma` should give the source distribution :math:`u` :
-    :math:`\sum_j \gamma_{ij} = u(x_i)` holds for all :math:`i`s and summing
+    :math:`\sum_j \gamma_{ij} = u(x_i)` holds for all :math:`i` s and summing
```

Comment:
It's correct without the"s", and it will render better.
```suggestion
    :math:`\sum_j \gamma_{ij} = u(x_i)` holds for all :math:`i` and summing
```

Reply:
I looked for references whether mathematical variable are plural or not but there seems are supports from both sides ([mathoverflow](https://mathoverflow.net/questions/192173/how-to-refer-to-plural-of-mathematical-symbols-with-or-without-an-apostrophe), [stackexchange](https://english.stackexchange.com/questions/337102/plural-for-math-notations), [stackexchange](https://ell.stackexchange.com/questions/167898/plural-of-a-mathematical-variable)). I personally feel like using plural forms of subscript variables is more natural, but I'm not native English speaking so I let you decide this one.

Reply:
> but I'm not native English speaking so I let you decide this one.

Thanks. "For all $x$" is a very common mathematical phrase, and "s" after the variable is uncommon in published work. I agree that with usual nouns we would need the "s" ("for all ducks...") but in this context I believe it is understood that we are talking about the _values_ that the variable can assume, e.g. "for all values of the variable $i$". Here are the first three academic texts about Wasserstein distance I found with the phrase "for all" (not selected to make a point), and "s" does not appear after the variable.

https://www.annualreviews.org/doi/full/10.1146/annurev-statistics-030718-104938
https://link.springer.com/article/10.1007/s13373-017-0101-1
https://library.oapen.org/bitstream/handle/20.500.12657/37704/1/2020_Book_AnInvitationToStatisticsInWass.pdf